STRUCTURE.md

```diff
@@ -14,11 +14,17 @@ See also [Deploying .natvis files](https://docs.microsoft.com/en-us/visualstudio
 
 ## The [`WinRT.Runtime`](/WinRT.Runtime) folder
 
-Contains the WinRT.Runtime project for building the C#/WinRT runtime assembly, winrt.runtime.dll.  There are two versions of this assembly, providing an abstraction layer over both .NET Standard 2.0 and .NET 5 runtimes.  While all code generated by cswinrt.exe is compatible with .NET Standard 2.0, there are runtime differences which must be addressed.  The primary difference is that the .NET 5 winrt.runtime.dll provides Xaml reference tracking support, which is necessary for WinUI 3 applications to manage memory correctly.  Future performance enhancements, such as function pointer support, may also be limited to the .NET 5 winrt.runtime.dll.
+Contains the WinRT.Runtime project for building the C#/WinRT runtime assembly, `WinRT.Runtime.dll`. The runtime assembly provides an abstraction layer over the .NET 5 runtime, and also provides Xaml reference tracking support which is necessary for WinUI 3 applications to manage memory correctly. The runtime assembly implements the following features for all projected C#/WinRT types:
+
+- WinRT activation and marshaling logic
+- Custom type mappings, primarily for WinUI
+- [CCW](https://docs.microsoft.com/dotnet/standard/native-interop/com-callable-wrapper) and [RCW](https://docs.microsoft.com/dotnet/standard/native-interop/runtime-callable-wrapper) management

```

Comment:
the CCW, RCW links contain a lot of obsolete information.  may want to avoid those specifically and just say "[COM Wrappers](https://docs.microsoft.com/en-us/dotnet/api/system.runtime.interopservices.comwrappers?view=net-5.0) management"